sensor/common/networkflow/manager/interfaces.go

```diff
@@ -13,4 +13,5 @@ type EntityStore interface {
 	LookupByEndpoint(net.NumericEndpoint) []clusterentities.LookupResult
 	RegisterPublicIPsListener(clusterentities.PublicIPsListener) bool
 	UnregisterPublicIPsListener(clusterentities.PublicIPsListener) bool
+	Tick()
```

Comment:
A more descriptive name might be good.

Reply:
The only thing that comes to my mind would be `RecordTick` - does that sound better?

Reply:
I assume that the new name is fine and resolve the conversation ;)  Let me know if you would see it under a different name